docs/src/token.md

```diff
@@ -247,6 +247,11 @@ Account's owner via the `Revoke` instruction.
 
 ### Multisignatures
 
+Warning: SPL Token v2's multisignature validation is compromised and should not
+be used.  Validation will erroneously pass a single valid signer signs m times.
+For more information:
+https://github.com/solana-labs/solana-program-library/issues/477
+
```

Comment:
I feel like we can do something strong directly in the v2 code as well and publish a new v2 crate.  Perhaps not this dramatic, but effectively an impossible modifications v2 to indicate clearly that v2 multisig is really Nx1

```diff
diff --git a/token/program/src/instruction.rs b/token/program/src/instruction.rs
index bd19a9e..84ba9f0 100644
--- a/token/program/src/instruction.rs
+++ b/token/program/src/instruction.rs
@@ -69,7 +69,7 @@ pub enum TokenInstruction {
     ///   0. `[writable]` The multisignature account to initialize.
     ///   1. `[]` Rent sysvar
     ///   2. ..2+N. `[]` The signer accounts, must equal to N where 1 <= N <= 11.
-    InitializeMultisig {
+    DangerInitializeMultisig {
         /// The number of signers (M) required to validate this multisignature account.
         m: u8,
     },
@@ -622,7 +622,7 @@ pub fn initialize_account(
 }
 
 /// Creates a `InitializeMultisig` instruction.
-pub fn initialize_multisig(
+pub fn danger_initialize_multisig(
     token_program_id: &Pubkey,
     multisig_pubkey: &Pubkey,
     signer_pubkeys: &[&Pubkey],
```

Reply:
Stronger language added